src/invertObject.js

```diff
@@ -19,7 +19,17 @@
  * @return {Object}
  */
 function invertObject(items) {
-  // write code here
+  const invertedObj = {};
+
+  for (const i in items) {
```

Comment:
`key` will be better
```suggestion
  for (const key in items) {
```